bundles/org.openhab.binding.androidtv/src/main/java/org/openhab/binding/androidtv/internal/AndroidTVBindingConstants.java

```diff
@@ -50,6 +50,8 @@ public class AndroidTVBindingConstants {
 
     // List of all config properties
     public static final String PROPERTY_IP_ADDRESS = "ipAddress";
+    public static final String PROPERTY_GOOGLETVPORT = "googletvPort";
+    public static final String PROPERTY_SHIELDTVPORT = "shieldtvPort";
```

Comment:
Naming: These are parameters, not properties.

Reply:
I updated all 4 in there that are actually parameters.